docker-alfresco/ais/docker-compose.yml

```diff
@@ -0,0 +1,173 @@
+version: "3"
+
+# The acs-deployment project contains the product version of docker-compose.
+#
+# This version is for testing and development use. For testing and debug we need to have a more open version.
+# For remote debug we need the CATALINA_OPTS and to expose port 8000.
+
+# Add the following environment variable to CATALINA_OPTS to activate YourKit profiling agent on tomcat
+#-agentpath:/usr/local/YourKit-JavaProfiler-2018.04/bin/linux-x86-64/libyjpagent.so=delay=200000,listen=all,sessionname=$$HOSTNAME,dir=/tmp/Alfresco/yourkit,onexit=snapshot,periodicperf=600,periodicmem=600,probe_off=*
+#
+# The identity service address should be the same as the origin of the request ( eg localhost ) for testing
+# So identity-service.auth-server-url=http://<IP>:8999 should point to the right address 
+# Otherwise, the requests to authenticate will fail
+
+services:
+    alfresco:
+        image: ${REGISTRY}alfresco/alfresco-content-repository:${ALFRESCO_TAG}
+        environment:
+            CATALINA_OPTS : "
+                -agentlib:jdwp=transport=dt_socket,address=*:8000,server=y,suspend=n
+                "
+            JAVA_OPTS : "
+                -Dalfresco.port=8082
+                -Daos.baseUrlOverwrite=http://localhost:8082/alfresco/aos
+                -Ddb.driver=org.postgresql.Driver
+                -Ddb.username=alfresco
+                -Ddb.password=alfresco
+                -Ddb.url=jdbc:postgresql://postgres:5432/alfresco
+                -Dsolr.host=solr6
+                -Dsolr.port=8983
+                -Dsolr.secureComms=none
+                -Dsolr.base.url=/solr
+                -Dindex.subsystem.name=solr6
+                -Dalfresco.restApi.basicAuthScheme=true
+                -Dimap.server.enabled=true
+                -Dftp.enabled=true
+                -Dftp.dataPortFrom=30000
+                -Dftp.dataPortTo=30099
+                -Dalfresco-pdf-renderer.url=http://alfresco-pdf-renderer:8090/
+                -Djodconverter.url=http://libreoffice:8090/
+                -Dimg.url=http://imagemagick:8090/
+                -Dtika.url=http://tika:8090/
+                -Dsfs.url=http://shared-file-store:8099/
+                -Dshare.host=localhost
+                -Dmessaging.broker.url=\"failover:(tcp://activemq:61616)?timeout=3000&jms.useCompression=true\"
+                -Dlocal.transform.service.enabled=${LOCAL_TRANSFORM_SERVICE_ENABLED}
+                -Dtransform.service.enabled=${TRANSFORM_SERVICE_ENABLED}
+                -Dauthentication.chain=identity-service1:identity-service,ldap:ldap,alfrescoNtlm1:alfrescoNtlm
+                -Didentity-service.enable-basic-auth=true
+                -Didentity-service.authentication.validation.failure.silent=false
+                -Didentity-service.auth-server-url=http://<HOST_IP>:8999/auth
```

Comment:
Do you not need to point to the created service alfresco-identity-service ?

Reply:
The HOST_IP is just a placeholder. I added a comment about this:

# The identity service address should be the same as the origin of the request ( eg localhost ) for testing
# So identity-service.auth-server-url=http://<IP>:8999 should point to the right address